plutus-core/plutus-core/stdlib/PlutusCore/StdLib/Meta/Data/Tuple.hs

```diff
@@ -28,11 +27,11 @@ import Control.Lens.Indexed (ifor, itraverse)
 import Data.Traversable
 
 -- | A Plutus Core (Scott-encoded) tuple.
-data Tuple term uni ann where
-    Tuple :: TermLike term TyName Name uni fun =>
```

Comment:
This seems suspiciously like a semantic change? cc @effectfully

Reply:
It was proposed by me to fix some inference issue with the newer version of GHC (I wonder if it's related to the disgusting abomination that quick look impredicativity is). It is a semantic change, but it doesn't seem to affect anything in our codebase and I'm almost sure it ended up there accidentally. `git blame` points to Mario.

Reply:
@effectfully is the one who suggested that change.

Reply:
:+1: